src/main/java/org/opensearch/searchrelevance/plugin/SearchRelevancePlugin.java

```diff
@@ -172,6 +200,15 @@ public Collection<Object> createComponents(
         this.clusterUtil = new ClusterUtil(clusterService);
         this.infoStatsManager = new InfoStatsManager(settingsAccessor);
         EventStatsManager.instance().initialize(settingsAccessor);
+        SearchRelevanceJobRunner jobRunner = SearchRelevanceJobRunner.getJobRunnerInstance();
+        jobRunner.setThreadPool(threadPool);
+        jobRunner.setClient(client);
+        jobRunner.setExperimentDao(experimentDao);
+        jobRunner.setQuerySetDao(querySetDao);
+        jobRunner.setSearchConfigurationDao(searchConfigurationDao);
+        jobRunner.setMetricsHelper(metricsHelper);
+        jobRunner.setHybridOptimizerExperimentProcessor(new HybridOptimizerExperimentProcessor(judgmentDao, experimentTaskManager));
```

Comment:
we aren't trying to schedule hybrid jobs, so not sure about the above line...   THough, how easy would it be to schedule a hybrid job?

Reply:
actually, since a hybrid job is stil lan experiment,t hen we can leave this!